docs/sources/mimir/release-notes/v2.15.md

```diff
@@ -0,0 +1,127 @@
+
+## Features and enhancements
+
+S2 compression for gRPC is now supported using the following flags:
+
+- `-alertmanager.alertmanager-client.grpc-compression=s2`
+- `-ingester.client.grpc-compression=s2`
+- `-querier.frontend-client.grpc-compression=s2`
+- `-querier.scheduler-client.grpc-compression=s2`
+- `-query-frontend.grpc-client-config.grpc-compression=s2`
+- `-query-scheduler.grpc-client-config.grpc-compression=s2`
+- `-ruler.client.grpc-compression=s2`
+- `-ruler.query-frontend.grpc-client-config.grpc-compression=s2`
+
+Distributors now support `lz4` OTLP compression, and can be deployed in multiple availability zones.
+
+The ruler's `<prometheus-http-prefix>/api/v1/rules` endpoint now supports `exclude_alerts`, `group_limit`, and `group_next_token` parameters.
+
+mimirtool's analyze ruler/prometheus commands now support bearer tokens.
+
+HTTP client settings can now be tuned for GCS and Azure backends via an `http` block or corresponding CLI flags.
+
+The compactor now refreshes deletion marks concurrently when updating the bucket index.
+
+The number of Memcached replicas for each type of cache can now be set using configuration settings when using jsonnet.
+
+## Important changes
+
+In Grafana Mimir 2.15, the following behavior has changed:
+
+The following alertmanager metrics are not exported for a `user` when the metric value is zero:
+
+- `cortex_alertmanager_alerts_received_total`
+- `cortex_alertmanager_alerts_invalid_total`
+- `cortex_alertmanager_partial_state_merges_total`
+- `cortex_alertmanager_partial_state_merges_failed_total`
+- `cortex_alertmanager_state_replication_total`
+- `cortex_alertmanager_state_replication_failed_total`
+- `cortex_alertmanager_alerts`
+- `cortex_alertmanager_silences`
+
+PromQL compatibility has been upgraded from Prometheus 2.0 to 3.0. More details can be found in the [Prometheus documentation](https://prometheus.io/docs/prometheus/3.0/migration/#promql). The following changes are of note:
+
+- The `.` pattern in regular expressions in PromQL now matches newline characters.
+- Lookback and range selectors are left open and right closed (previously left closed and right closed).
+- Native histograms now use exponential interpolation.
+
+Backwards compatibility in dashboards and alerts for `thanos_memcached_`-prefixed metrics has been removed. These metrics were removed in 2.12 in favor of `thanos_cache_`-prefixed metrics.
+
+Support for Redis as a cache backend has been removed from jsonnet.
+
+The following deprecated configuration options were removed in this release:
+
+- `-distributor.direct-otlp-translation-enabled`, which has been enabled by default since 2.13 and is now considered stable.
+- `-query-scheduler.prioritize-query-components`, which is always enabled now
+- `-ingest-storage.kafka.ongoing-records-per-fetch` and `-ingest-storage.kafka.startup-records-per-fetch`, which have been removed in favour of `-ingest-storage.kafka.max-buffered-bytes`
```

Comment:
Everything about `ingest-storage` is still experimental and we skip mentioning it. I think, we can drop this one from the notes:

```suggestion
```